src/index.html

```diff
@@ -11,12 +11,106 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          class="img"
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav_link">
+        <ul class="ul_link">
+          <li>
+            <a
+              href="#"
+              class="is-active a_link"
+            >
+              Apple
+            </a>
+          </li>
```

Comment:
let's change class naming here, you'll learn more about it in the next `BEM` topick

`nav_link` -> `nav`
`ul_link` -> `nav__list`
`left-space` -> `nav__item`
`a_link` -> `nav__link`